pages/canvas.tsx

```diff
@@ -16,7 +18,11 @@ export default function Canvas() {
         <Drawing />
         <div className={styles.footercontainer}>
           <Button className={styles.homebtn} label="Home" />
-          <Button className={styles.backbtn} label="Back" />
+          <Button
+            className={styles.backbtn}
+            onClick={router.back}
+            label="Back"
+          />
```

Comment:
Instead of using `useRouter` it might be better to use React Link components, if the button only redirects to a specific page.
```suggestion
<Link>
<a href="/">
          <Button
            className={styles.backbtn}
            label="Back"
          />
</a>
</Link>
```